website/replay.php

```diff
@@ -61,12 +61,17 @@ function poll_as_replay() {
 var g_remote_poller;
 var g_recorder;
 
-var g_replay_count;
-var g_replay_rate;
+var g_replay_duration = <?php echo read_raceinfo('replay-skipback', '4'); ?>;
```

Comment:
I don't think there's a good way forward here, given the state of the code and the fact that the replay protocol doesn't tell you the skip-back value until the moment of playback.  

I was going to suggest making this a control on the replay page, but that still makes it not easy to adjust after it's turned on.

I'm hoping that the whole VideoCaptureFlight mechanism can be scrapped in favor of a simpler frame buffer strategy, and the replay protocol clearly needs a re-think.

I wonder if it wouldn't be better overall for you to just tailor the code locally to your specific case, and hold off trying to integrate a partial solution on a not-great underlying platform?

Reply:
Thanks for the feedback. I have added values I wanted to see for replay and force reloaded the page to make it work for my case.

At this point, since there are issues with this, might be best to just close this PR without accepting it and revisit the whole thing later.